docs/source/search-syntax.rst

```diff
@@ -198,7 +198,7 @@ To get all runs from the experiment named "Social NLP Experiments", use:
 
   import mlflow
 
-  runs = mlflow.search_runs(experiment_name=["Social NLP Experiments"])
+  runs = mlflow.search_runs(experiment_names=["Social NLP Experiments"])
```

Comment:
Thanks for the catch!

Reply:
Welcome. Thanks for making/maintaining a great package :)